modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -733,11 +733,13 @@ public void preprocessOpenAPI(OpenAPI openAPI) {
                             schemas.put(opId, requestSchema);
                         }
                         // process all response bodies
-                        for (Map.Entry<String, ApiResponse> ar : op.getValue().getResponses().entrySet()) {
-                            ApiResponse a = ModelUtils.getReferencedApiResponse(openAPI, ar.getValue());
-                            Schema responseSchema = ModelUtils.getSchemaFromResponse(a);
-                            if (responseSchema != null) {
-                                schemas.put(opId + ar.getKey(), responseSchema);
+                        if (op.getValue().getResponses() != null){
```

Comment:
👍

Reply:
Same here as in Line 713.
Here are following test failed with NPE in SpringCodegenTest:
doGenerateCookieParams
doGenerateRequestParamForSimpleParam
doNotGenerateRequestParamForObjectQueryParam
generateFormatForDateAndDateTimeQueryParam